src/cart/crt_iv.c

```diff
@@ -1628,7 +1632,7 @@ crt_iv_fetch(crt_iv_namespace_t ivns, uint32_t class_id,
 		iv_ops->ivo_on_refresh(ivns_internal, iv_key, 0,
 				NULL, false, rc, user_priv);
 		fetch_comp_cb(ivns_internal, class_id, iv_key, NULL,
-			      NULL, rc, cb_arg);
+			      iv_value, rc, cb_arg);
```

Comment:
same comment as above. instead of using iv_value as 'misc_value' in error case, a new arg to fetch_comp_cb() should be added instead